src/poliastro/constants.py

```diff
@@ -335,3 +335,13 @@
     "Howard Curtis",
     system="si",
 )
+
+altitude_GEO = Constant(
```

Comment:
Bear in mind that GEO altitude depends on the planet, so I would prefer not to include it as a constant.

Reply:
Yes, to make it work for any given attractor, that cannot be a constant. Will work on it.